sql/pre_install/tables.sql

```diff
@@ -276,6 +276,8 @@ CREATE TABLE _timescaledb_config.bgw_job (
   proc_name name NOT NULL,
   owner name NOT NULL DEFAULT CURRENT_ROLE,
   scheduled bool NOT NULL DEFAULT TRUE,
+  fixed_schedule bool not null default true,
+  initial_start timestamptz,
   hypertable_id integer,
```

Comment:
I put them in the middle so that I could still use `GETSTRUCT` for the fixed_schedule, (since it would need to come before the config which is variable length).